src/components/Popups/__tests__/PopupReply-test.tsx

```diff
@@ -28,11 +40,12 @@ describe('components/Popups/PopupReply', () => {
         preventDefault: jest.fn(),
         stopPropagation: jest.fn(),
     };
-    const mockTextarea = { focus: jest.fn(), value: '' };
+    const mockEditor = { focus: jest.fn() };
```

Comment:
Should we mock out the editor via `jest.mock`? It doesn't look like PopupReply deals with focus anymore.